news/2024/2024-11-30-new-featured-artist-zmey-gorynich.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "New Featured Artist: Zmey Gorynich"
+date: 2024-11-30 20:00:00 +0000
+---
+
+A new species of metal music crawled into our Featured Artist library, courtesy of **Zmey Gorynich**!
+
+![](https://assets.ppy.sh/artists/450/header.jpg)
+
+Self-proclaimed "buffoons", Russian band **Zmey Gorynich** made a name for themselves by playing an usual combo of cheerful extreme metal since 2012. They were brought to the forefront of osu! by metal mastermind [Mazzerin](https://osu.ppy.sh/users/2942381) with [Sorochinskaya Yarmarka](https://osu.ppy.sh/beatmapsets/1191853), a stream map featuring unreal slider-centric design perfectly fitting the insanity of the tune.
```

Comment:
"band" is singular
guesswork on my part for the ambiguous typo

```suggestion
Self-proclaimed "buffoons", Russian band **Zmey Gorynich** made a name for itself by playing an unusual combo of cheerful extreme metal since 2012. They were brought to the forefront of osu! by metal mastermind [Mazzerin](https://osu.ppy.sh/users/2942381) with [Sorochinskaya Yarmarka](https://osu.ppy.sh/beatmapsets/1191853), a stream map featuring unreal slider-centric design perfectly fitting the insanity of the tune.
```